distributed/deploy/ssh.py

```diff
@@ -89,7 +89,19 @@ def __init__(
         self.kwargs = copy.copy(kwargs)
         self.name = name
         self.remote_python = remote_python
-        self.nprocs = self.kwargs.pop("nprocs", 1)
+        if kwargs.get("nprocs") is not None and kwargs.get("n_workers") is not None:
+            raise ValueError(
+                "Both nprocs and n_workers were specified. Use n_workers only."
+            )
+        elif kwargs.get("nprocs") is not None:
+            warnings.warn(
+                "The nprocs argument will be removed in a future release. It has been "
+                "renamed to n_workers.",
+                FutureWarning,
+            )
+            self.n_workers = self.kwargs.pop("nprocs")
+        else:
+            self.n_workers = self.kwargs.pop("n_workers", 1)
```

Comment:
Similar comment here about adding a `self.nprocs` property redirecting to `self.n_workers`

Reply:
Similar response as above 😄